source/fundamentals/data-formats/document-data-format-data-class.txt

```diff
@@ -114,6 +114,13 @@ You can use the following annotations on data classes:
    * - ``BsonRepresentation``
      - Specifies the BSON type used to store the value when different from the
        data class property.
+       
+       .. note::
+          
+          Use this annotation only on a property you are
+          storing as a different BSON type. You might experience errors
+          if you include the ``BsonRepresentation`` annotation on a
+          property you are storing as the same type.
```

Comment:
Suggestion:

I think the current description ("Specifies the BSON type...") could be modified to communicate what's in this callout which would make the content of this description less repetitive.

"Specifies the BSON type MongoDB uses to store the value. Use this annotation only when you need to store the value as a BSON type that differs from the data class property."

And perhaps the callout could be a warning if necessary:

"Your code may throw exceptions If you specify the same type as the data class."